navigator/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>molgenis-apps-core</artifactId>
+        <groupId>org.molgenis</groupId>
+        <version>1.0.0-SNAPSHOT</version>
```

Comment:
We need something like "1.0.0-APPS-SNAPSHOT" to distinguish between the 1.0.0 release of the molgenis/molgenis repo and this new apps repo